code/modules/media/broadcast/transmitters/broadcast.dm

```diff
@@ -95,8 +95,6 @@
 
 /obj/machinery/media/transmitter/broadcast/attackby(var/obj/item/W, mob/user)
 	. = ..()
-	if(.)
```

Comment:
If someone puts it in their hand, so its not as big of a concern as it would be if you modified attack_hand instead